samples/tab-calendar/ts/Manifest/manifest.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.16/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.16",
+  "version": "1.0.0",
+  "id": "{{Microsoft-App-Id}}",
+  "packageName": "com.teams.TabCalendar",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://www.microsoft.com",
+    "privacyUrl": "https://www.teams.com/privacy",
+    "termsOfUseUrl": "https://www.teams.com/termsofuser"
+  },
+  "icons": {
+    "color": "color.png",
+    "outline": "outline.png"
+  },
+  "name": {
+    "short": "Tab Calendar",
+    "full": "Tab Calendar"
+  },
+  "description": {
+    "short": "Short description for Tab Calendar",
```

Comment:
Interact with Calendar in Outlook from Personal Tab

Reply:
Fixed